tools/convert_datasets/a2d2.py

```diff
@@ -0,0 +1,389 @@
+    (0, 150, 0):      17,  # Small vehicles 3 --> Motorcycle ?
+    (0, 128, 255):     6,  # Traffic signal 1 --> Traffic light
+    (30, 28, 158):     6,  # Traffic signal 2 --> Traffic light
+    (60, 28, 100):     6,  # Traffic signal 3 --> Traffic light
+    (0, 255, 255):     7,  # Traffic sign 1 --> Traffic sign
+    (30, 220, 220):    7,  # Traffic sign 2 --> Traffic sign
+    (60, 157, 199):    7,  # Traffic sign 3 --> Traffic sign
+    (255, 255, 0):    13,  # Utility vehicle 1 --> Car ?
+    (255, 255, 200):  13,  # Utility vehicle 2 --> Car ?
+    (233, 100, 0):     4,  # Sidebars --> Fence
+    (110, 110, 0):     0,  # Speed bumper --> Road
+    (128, 128, 0):     1,  # Curbstone --> Sidewalk
+    (255, 193, 37):    0,  # Solid line --> Road
+    (64, 0, 64):     255,  # Irrelevant signs --> Ignore
+    (185, 122, 87):    4,  # Road blocks --> Fence
+    (139, 99, 108):    0,  # Non-drivable street --> Road
+    (210, 50, 115):    0,  # Zebra crossing --> Road
+    (255, 0, 128):   255,  # Obstacles / trash --> Ignore ?
+    (255, 246, 143):   5,  # Poles --> Poles
+    (150, 0, 150):     0,  # RD restricted area --> Road ?
+    (204, 255, 153): 255,  # Animals --> Ignore ?
+    (238, 162, 173):   0,  # Grid structure --> Road ???
+    (33, 44, 177):     6,  # Signal corpus --> Traffic light ?
+    (180, 50, 180):    0,  # Drivable cobblestone --> Road
+    (255, 70, 185):  255,  # Electronic traffic --> Ignore ???
+    (238, 233, 191):   0,  # Slow drive area --> Road
+    (147, 253, 194):   8,  # Nature object --> Vegetation
+    (150, 150, 200):   0,  # Parking area --> Road
+    (180, 150, 200):   1,  # Sidewalk --> Sidewalk
+    (72, 209, 204):  255,  # Ego car --> Ignore
+    (200, 125, 210):   0,  # Painted driv. instr. --> Road ???
+    (159, 121, 238):   0,  # Traffic guide obj. --> Road ???
+    (128, 0, 255):     0,  # Dashed line --> Road
+    (255, 0, 255):     0,  # RD normal street --> Road
+    (135, 206, 255):  10,  # Sky --> Sky
+    (241, 230, 255):   2,  # Buildings --> Building
+    (96, 69, 143):   255,  # Blurred area --> Ignore ?
+    (53, 46, 82):    255,  # Rain dirt --> Ignore ?
+}
+
+
+def modify_label_filename(label_filepath):
+    ''' Returns a mmsegmentation-combatible label filename.
+    '''
+    label_filepath = label_filepath.replace('_label_', '_camera_')
+    label_filepath = label_filepath.replace('.png', '_labelTrainIds.png')
+    return label_filepath
+
+
+def convert_a2d2_trainids(label_filepath, ignore_id=255):
+    '''Saves a new semantic label using the A2D2 label categories.
+
+    The new image is saved into the same directory as the original image having
+    an additional suffix.
+
+    Args:
+        label_filepath: Path to the original semantic label.
+        ignore_id: Default value for unlabeled elements.
+    '''
+    # Read label file as Numpy array (H, W, 3)
+    orig_label = Image.open(label_filepath)
+    orig_label = np.array(orig_label, dtype=np.int)
```

Comment:
`mmcv.imread`

Reply:
Changed to `orig_label = mmcv.imread(label_filepath)`